driver/remote/driver.go

```diff
@@ -90,7 +90,7 @@ func (d *Driver) Client(ctx context.Context) (*client.Client, error) {
 func (d *Driver) Features() map[driver.Feature]bool {
 	return map[driver.Feature]bool{
 		driver.OCIExporter:    true,
-		driver.DockerExporter: false,
+		driver.DockerExporter: true,
```

Comment:
I don't understand why this was false.

Reply:
:eyes: yeah, I'm not sure either. Looks like it didn't get caught in https://github.com/docker/buildx/pull/1078, my bad.